core/src/main/java/feast/core/config/FeastProperties.java

```diff
@@ -72,11 +73,19 @@ public FeastProperties() {}
   /* Feast Kafka stream properties */
   private StreamProperties stream;
 
-  private SecurityProperties security;
+  @NotNull private SecurityProperties security;
 
   @Bean
   SecurityProperties securityProperties() {
-    return this.getSecurity();
+    return getSecurity();
+  }
+
+  /* Feast Audit Logging properties */
+  @NotNull private LoggingProperties logging;
+
+  @Bean
```

Comment:
I know you are modeling this after the SecurityProperties, but do we really need Beans here? For example, why does JobProperties work?

Reply:
See above comment about consolidating behind `FeastProperties`